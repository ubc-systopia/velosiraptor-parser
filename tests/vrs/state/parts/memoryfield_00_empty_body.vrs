mem foo [ base, 8, 0 ] {}