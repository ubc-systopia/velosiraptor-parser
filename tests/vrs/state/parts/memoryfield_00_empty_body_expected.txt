error: empty state field block
     --> tests/vrs/state/parts/memoryfield_00_empty_body.vrs:1:26
      |
    1 |  mem foo [ base, 8, 0 ] {}
      |                            remove the block or fill it in explicitly
